x-pack/plugins/remote_clusters/public/sections/components/remote_cluster_form/validators/validate_seed.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+import {
+  isSeedNodeValid,
+  isSeedNodePortValid,
+} from '../../../../services';
+
+export function validateSeed(seed) {
+  const errors = [];
+
+  if (!seed) {
+    return errors;
+  }
+
+  const isInvalid = !isSeedNodeValid(seed);
```

Comment:
The positive is clearer to me: `const isValid = isSeedNodeValid(seed)` and then `if (!isValid) ...`